### ISSUE TITLE
Standard Petri net semantics are not preserved in SVIIvR model for susceptible->exposed

### ISSUE THREAD

**Issue by djinnome** [indralab/mira#120]
Hi folks,

The semantics for Petri nets is that tokens must be present in all places that point to a transition in order for it to fire.

However, in the Petri net
```
model_2 = stratify(
    model_1, 
    key="vaccination_status", 
    strata={"vaccinated", "unvaccinated"},
    structure=[("unvaccinated", "vaccinated")],
    directed=True,
    cartesian_control=True,
    # TODO add exclusion
    # exclude=[dead],
)
```

there are three edges pointing into the transition from susceptible vaccinated to exposed.  The other two edges are vaccinated infected AND unvaccinated infected.   Thus the interpretation is that a succeptible vaccinated person needs to be exposed to BOTH a vaccinated infected person AND an unvaccinated infected person. It would adhere to standard Petri net semantics if there were separate transitions for infected vaccinated and infected unvaccinated.

It seems you want to think of infected vaccinated and infected unvaccinated as controllers that are neither created or destroyed in a conversion, in which case, the semantics of multiple controllers is OR, rather than AND.

**Comment by bgyori** [comment]
Thanks, I'll  try to fix this tomorrow!

**Comment by bgyori** [comment]
This is solved in #125
